spec/javascripts/components/toggle-spec.js

```diff
@@ -59,6 +65,39 @@ describe('A toggle module', function () {
       expect(element.find('.my-toggle').data('toggled-text')).toBe('Show fewer');
       expect(element.find('.my-toggle').text()).toBe('Toggle');
     });
+
+    it('does not track when element is not trackable', function() {
+      spyOn(GOVUK.analytics, 'trackEvent');
+
+      element = $('\
+        <div>\
+          <a href="#" class="my-toggle" data-expanded="false" data-controls="target" data-toggled-text="Show fewer">Toggle</a>\
+          <div id="target" class="js-hidden">Target</div>\
+        </div>');
```

Comment:
This seems to be creating an element similar to the one already created in the `beforeEach` function (which has also been modified in this PR). Do we have confidence that this test only has one toggle element?

Reply:
Yes, because we're overriding `element` and creating a ` new GOVUK.Modules.GemToggle()` that is overriding `toggle`. Happy to add another clause in there to prove this if you feel it's required.

Reply:
Oh I see now. Thanks.